packages/base/core/src/less/_variables.less

```diff
@@ -1817,11 +1817,11 @@
 @oui-spinner-l-speed-scale: 240ms;
 
 // Status
-@oui-status-line-height: 1.375;
+@oui-status-line-height: 1;
 @oui-status-max-width: rem-calc(200);
-@oui-status-padding: rem-calc(4) rem-calc(10);
-@oui-status-font-color: @oui-color-sapphire;
-@oui-status-font-color_info: @info-color;
+@oui-status-padding: rem-calc(2) rem-calc(8) ;
+@oui-status-font-color: @text-color;
+@oui-status-font-color_info: @text-color;
```

Comment:
I think we should keep `@info-color` for that. If this color changes, it will take automatically the new color.
If `@info-color` haven't the right color, maybe it should be this value to change (from what I see of the messages style in the mockup) :)